benchmark/Streamly/Benchmark/Data/Array/Stream.hs

```diff
@@ -285,15 +285,17 @@ main = do
     where
 
     alloc value = do
-        small <- Stream.toList $ Stream.arraysOf 100 $ sourceUnfoldrM value 0
-        big <- Stream.toList $ Stream.arraysOf value $ sourceUnfoldrM value 0
+        let val = max value 1
+        small <- Stream.toList $ Stream.arraysOf 100 $ sourceUnfoldrM val 0
+        big <- Stream.toList $ Stream.arraysOf val $ sourceUnfoldrM val 0
```

Comment:
Only `runWithCLIOptsEnv` calls this with value `0`. And it perhaps does not use the return value of alloc. So we can just try returning (undefined, undefined) if the value passed is 0.